src/utils.js

```diff
@@ -64,6 +64,27 @@ export const indexOf = (array, value) => {
   return -1;
 };
 
+/**
+ * @function lastIndexOf
+ *
+ * @description
+ * get the last index of the value in the array
+ * lastIndexOf is usefull where the array is stack like
+ *
+ * @param {Array<any>} array the array to get the index of the value at
+ * @param {any} value the value to match
+ * @returns {number} the index of the value in array
+ */
+const lastIndexOf = (array, value) => {
+  for (var index = array.length; index-- > 0;) {
```

Comment:
did u run this locally? `var` statements are not allowed by the linting rules, this should have blown up.